VERSION

```diff
@@ -1 +1 @@
-1.1.2
```

Comment:
what's this?

Reply:
This file is used to override the value of a `version` constant in https://github.com/argoproj/argo-cd/blob/master/common/version.go#L11